plugins/kubesvc/patch-clouddriver.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: spinnaker.armory.io/v1alpha2
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  spinnakerConfig:
+    profiles:
+      clouddriver:
+        kubernetes:
```

Comment:
I'd mention this is not necessary if you already have v2 accounts defined (and enabled!).